cross/ffmpeg/Makefile

```diff
@@ -24,7 +24,7 @@ LICENSE  = GPLv2
 
 NASM_PATH=$(WORK_DIR)/../../../native/nasm/work-native/install/usr/local/bin
 
-CONFIGURE_ARGS = --target-os=linux --cross-prefix=$(TC_PATH)$(TC_PREFIX) --prefix=$(INSTALL_PREFIX)
+CONFIGURE_ARGS = --target-os=linux --cross-prefix=$(TC_PATH)$(TC_PREFIX)- --prefix=$(INSTALL_PREFIX)
```

Comment:
I would avoid these TC_PREFIX "meaning" changes... why not keep dash in variable value.

Reply:
Excellent catch.  I'll come up with another approach.

Reply:
As always your comments are valuable, thnx @ymartin59 .
On this, I've reverted to the previous behavior and re-declared `TC_TARGET`.
If `TC_PREFIX` is not set it becomes `TC_PREFIX = $(TC_TARGET)-` ... 
Note that it is never manually set in any of the `toolchains/*/Makefile`
And this avoids all the small changes to reflect the missing `-` reverting back multiple files to its original state.